items.json

```diff
@@ -6460,6 +6460,74 @@
       }
     ]
   },
+  {
+    "id": "crpg_broad_arrows_a",
+    "name": "Broad Head Arrows",
+    "culture": "Neutral",
+    "type": "Arrows",
+    "price": 695,
+    "weight": 0.098,
+    "tier": 3.22210383,
+    "requirement": 0,
+    "flags": [],
+    "weapons": [
+      {
+        "class": "Arrow",
+        "itemUsage": "arrow_right",
+        "accuracy": 100,
+        "missileSpeed": 10,
+        "stackAmount": 22,
+        "length": 97,
+        "balance": 0.0,
+        "handling": 0,
+        "bodyArmor": 0,
+        "flags": [
+          "Consumable",
+          "BonusAgainstShield"

```

Comment:
It feels like you're breaking the natural balance of things. This should be discussed with other people. I'm not sure it is the role of archer to break shields. This is supposed to be axes roles. Also i'm not even sure adding this flags works. In my memory there is no bonusagainstshield checking when computing arrow damage over shield. I do like the idea of cut arrows. I'd remove the bonus against shield and buff the cut damage on arrows up to 10

Reply:
Aye Aye Captain